src/client/cassh

```diff
@@ -204,7 +204,7 @@ class CASSH(object):
         except ConnectionError:
             print('Connection error : %s' % self.url)
             exit(1)
-        if not 'ssh-' in req.text:
+        if not 'ssh-' in req.text and not 'ecdsa-' in req.text:
```

Comment:
are you sure about that change ? ecdsa is in fact a type of ssh key, like rsa, dsa... so I find this condition quite weird

Reply:
```
ecdsa-sha2-nistp521 AAAAE2VjZHNhLXNoYTItbmlzdHA1MjEAAAAIbmlzdHA1MjEAAACFBACf61W//oAY/pDJh7t4Nx2/GKfq42S3FGacxVoWGs4hBCtCzTnPKS3pFIvorehSccE7RN5fxjGqg8FAz/ktJlmhdQC6Nzx6UD+k3iKZgrcoI/RXIm9Hw7HB
7UNb6W+EmDGbOtj+DH0bBu73XG0KZmFNqaZPAJzcgbq7U09+01dKlBbWRA== example2
```

Reply:
I should add also ed25519, but maybe another time, because I need to do some test

Reply:
It's `ssh-ed25519`, so it's already working